pkg/util/nsenter/nsenter.go

```diff
@@ -83,8 +84,11 @@ func NewNsenter() *Nsenter {
 	}
 	// search for the required commands in other locations besides /usr/bin
 	for binary := range ne.paths {
-		// default to root
-		ne.paths[binary] = filepath.Join("/", binary)
+		// check in root directory first
+		binPath := filepath.Join("/", binary)
```

Comment:
why not put "/" into `range []string{"/bin", "/usr/sbin", "/usr/bin"}`, it could make code more clear

Reply:
good point. done!